daiquiri/core/management/commands/delete_users.py

```diff
@@ -0,0 +1,61 @@
+import re
+import sys
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument(
```

Comment:
doc says: https://docs.python.org/3/library/argparse.html#required

If argname starts with `-` or `--` then default is `required=False` otherwise default is `required=True`. Therefore since argname is `id_list_file`, `required=False` is not required.